### PR TITLE
command usage in help plugin

### PR DESCRIPTION
**Pull request by hagane**
So, i've got around to #370
Can someone please review this?

### CODE REVIEW

**Comment by ForNeVeR** [comment]
I'm on it.

**Comment by ForNeVeR** [comment]
I guess we should now migrate all the commands' help localization to the new key format?

**Comment by hagane** [comment]
Wait, disregard that, I've messed up the commands (`say` instead of `send`) (%
I may as well write all the command usages while fixing that.

**Comment by hagane** [comment]
There, I fixed it. Enjoy.

File: src/main/resources/localization/en.conf [hunk 1]
```diff
@@ -124,3 +124,16 @@ CommonAccess: "Common"
...
   "%s is barely breathing, letting out husky sounds and coughing up bloody foam."
 ]
+
+// Command usage:
+"command.help.bash": "Usage: $bash: A random quote from http://bash.im."
+"command.help.dice": "Usage: $dice [faces] [count]: roll a [count] of [faces]-sided dice."
+"command.help.help": "Usage: $help: display available commands. | $help [command]: dislay command description."
+"command.help.loglist": "Usage: $loglist [quoteId]: a quote from http://loglist.net. Gives a random quote if no quoteId is given."
+"command.help.pet": "See $pet help."
+"command.help.s": "Usage: $s |$s [number]: shorthand for $say"
```
Comment (ForNeVeR) [inline src/main/resources/localization/en.conf]: Dat is not right. I'll fix it myself, gimme a minute.

Comment (hagane) [inline src/main/resources/localization/en.conf]: The funny thing is that I've known about the replace command, but seeing it in Markov plugin made me have a brain fart -_\

**Comment by ForNeVeR** [comment]
LGTM :+1:

I'll now make some manual tests and maybe merge that. Thanks for the contribution.

File: src/main/resources/localization/en.conf (hunk 1 above)
Comment (ForNeVeR) [inline src/main/resources/localization/en.conf]: That's a historical artifact and sooner or later we'll get rid of that: #133.

**Pull request merged**